apply/run.go

```diff
@@ -32,8 +32,10 @@ func IsIPList(args string) bool {
 	ipList := strings.Split(args, ",")
 
 	for _, i := range ipList {
-		ip := net.ParseIP(i)
-		if ip == nil {
+		if !strings.Contains(i, ":") {
+			return net.ParseIP(i) != nil
+		}
+		if _, err := net.ResolveTCPAddr("tcp", i); err != nil {
```

Comment:
LGTM.
Curious about ResolveTCPAddr. The reason for resolve tcp is that ssh uses tcp, isn't it?